src/dom/dom.js

```diff
@@ -3325,7 +3325,7 @@ var Cue = function(callback, time, id, val) {
  * //
  * //
  * function setup() {
- *   noCanvas();
+ *   createCanvas(200,200);
```

Comment:
```suggestion
 *   createCanvas(200, 200);
```

This should make the linter pass.